app/views/users/_header.html.erb

```diff
@@ -0,0 +1,33 @@
+<nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+  <div class="container-fluid">
+
+    <a class="navbar-brand" href="#">
+    <!-- we need an image to place here -->
+      <img src="https://w7.pngwing.com/pngs/791/261/png-transparent-black-spade-spades-suit-playing-card-shovel-ace-card-leaf-silhouette-garden-thumbnail.png" alt="" width="30" height="40">
+    </a>
+
+    <%= link_to "Acebook", root_path, class:"navbar-brand"%>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+        <% if logged_in? %>
+          <li class="nav-item">
+            <%= link_to "Add Post", new_post_path, class:"nav-link"%>
+          </li>
+          <li class="nav-item">
+            <%= link_to "Sign-out", logout_path, method: :delete, class:"nav-link" %>
+          </li>
+        <% else %>
+          <li class="nav-item">
+          <%= link_to "Login", login_path, class:"nav-link" %>
+          </li>
+          <li class="nav-item">
+          <%= link_to "sign up", signup_path, class:"nav-link" %>
```

Comment:
the "Sign-out", "Login", and "sign up" on lines 20, 24 and 27 are what's visible on the UI right? If so they should follow the same format. I think they should be "Logout", "Login" and "Sign up" instead

Reply:
Agreed